### PR TITLE
Display error for boolean fields

### PR DESCRIPTION
**Pull request by nixocio**
Display error when modifying Allow Override Branch on Projects.

See: https://github.com/ansible/awx/issues/10144

image

### CODE REVIEW

**Comment by nixocio** [comment]
I triggered an e2e test run.

File: awx/ui_next/src/components/FormField/sortErrorMessages.js [hunk 1]
```diff
@@ -52,6 +52,9 @@ function parseFieldErrors(obj, formValues) {
       }
       formErrors = formErrors.concat(parsed.formErrors);
     }
+    if (typeof formValues[key] === 'boolean') {
```
Comment (keithjgrant) [inline awx/ui_next/src/components/FormField/sortErrorMessages.js:55]: I wonder if this should just be an `else` block to catch any and all "other" types that could occur

Comment (nixocio) [reply awx/ui_next/src/components/FormField/sortErrorMessages.js:55]: I tried the catch all - else, but the error was not displayed.

**Review by keithjgrant — APPROVED** [review]

**Comment by akus062381** [comment]
@nixocio this is checked and verified, ok as far as QE is concerned!

**Pull request merged**